torchmetrics/image/psnr.py

```diff
@@ -110,7 +110,7 @@ def __init__(
         self.reduction = reduction
         self.dim = tuple(dim) if isinstance(dim, Sequence) else dim
 
-    def update(self, preds: Tensor, target: Tensor) -> None:
+    def update(self, preds: Tensor, target: Tensor) -> None:  # type: ignore
```

Comment:
this is the same as in #332 right? So contradicting to what I wrote there maybe merge them and do a PR with a followup on all signatures if we found a solution?

Reply:
yes, want to make another PR to try to resolve this typing over all codebase